examples/medplum-nextjs-demo/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "medplum-nextjs-demo",
+  "version": "2.0.26",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@emotion/react": "11.11.1",
+    "@emotion/server": "11.11.0",
+    "@mantine/core": "6.0.17",
```

Comment:
`@mantine` dependences in `packages/app` and `packages/react` are all on `6.0.16` - can we align for now?  We'll upgrade them all at once in the next dependency upgrade.